webviz_config/_build_webviz.py

```diff
@@ -22,22 +24,22 @@ def build_webviz(args: argparse.Namespace) -> None:
         raise ValueError(f"Theme `{args.theme}` is not installed.")
 
     if args.portable is None:
-        build_directory = tempfile.mkdtemp()
+        build_directory = pathlib.Path(tempfile.mkdtemp()).resolve().parent
```

Comment:
`tempfile.mkdtemp()` already returns an absolute path, so don't need to call `resolve()`. Also you want the folder itself, and not the parent folder.

```suggestion
        build_directory = pathlib.Path(tempfile.mkdtemp())
```